tests/project.go

```diff
@@ -43,3 +43,108 @@ func TestCreateProject(t *testing.T) {
 	}
 	assert.True(t, sawNewProject)
 }
+
+func TestUpdateProjectDescription(t *testing.T) {
+	ctx := context.Background()
+	client, conn := GetTestAdminServiceClient()
+	defer conn.Close()
+
+	// Create a new project.
+	req := admin.ProjectRegisterRequest{
+		Project: &admin.Project{
+			Id:   "potato",
+			Name: "spud",
+			Labels: &admin.Labels{
+				Values: map[string]string{
+					"foo": "bar",
+					"bar": "baz",
+				},
+			},
+		},
+	}
+	_, err := client.RegisterProject(ctx, &req)
+	assert.Nil(t, err)
+
+	// Verify the project has been registered.
+	projects, err := client.ListProjects(ctx, &admin.ProjectListRequest{})
+	assert.Nil(t, err)
+	assert.NotEmpty(t, projects.Projects)
+
+	// Attempt to modify the name of the Project. Modifying the Name should be a
+	// no-op, while the Description is modified. Labels should be a no-op.
+	_, err = client.UpdateProject(ctx, &admin.Project{
+		Id: "potato",
+		Name: "foobar",
+		Description: "a-new-description",
+		Labels: &admin.Labels{},
```

Comment:
@katrogan — according to our hypothesis about how gorm works, the update with empty labels should be a no-op.